smac/model/random_forest/random_forest.py

```diff
@@ -52,19 +633,33 @@ class RandomForest(AbstractRandomForest):
     def __init__(
         self,
         configspace: ConfigurationSpace,
-        n_trees: int = N_TREES,
         n_points_per_tree: int = -1,
         ratio_features: float = 5.0 / 6.0,
-        min_samples_split: int = 3,
-        min_samples_leaf: int = 3,
-        max_depth: int = 2**20,
         eps_purity: float = 1e-8,
-        max_nodes: int = 2**20,
-        bootstrapping: bool = True,
         log_y: bool = False,
         instance_features: dict[str, list[int | float]] | None = None,
         pca_components: int | None = 7,
         seed: int = 0,
+        n_trees: int = N_TREES,  # TODO: HP: 100 is default in sklearn, here it's 10
+        cross_trees_variance: bool = False,
+        criterion: str = "squared_error",
+        splitter: str = "random",  # Should not be changed
+        max_depth: int = 2**20,  # TODO: HP: None is default in sklearn, here it's 2**20
```

Comment:
rewrite comments, afaik there is no TODO. these were the HPs of the previous RF, right?

Reply:
in all cases, keep the information somewhere